pkgs/tools/networking/persepolis/default.nix

```diff
@@ -26,10 +30,25 @@ buildPythonApplication rec {
   # see: https://github.com/persepolisdm/persepolis/blob/3.2.0/setup.py#L130
   doCheck = false;
 
-  preBuild=''
+  preBuild=
+  # Make setup automatic
+  ''
     substituteInPlace setup.py --replace "answer = input(" "answer = 'y'#"
+  '' +
+  # Replace abandoned youtube-dl with maintined for yt-dlp. Fixes https://github.com/persepolisdm/persepolis/issues/930,
```

Comment:
```suggestion
  # Replace abandoned youtube-dl with yt-dlp. Fixes https://github.com/persepolisdm/persepolis/issues/930,
```

Reply:
Ah that's a typo, I wanted to write "maintained fork" to signify that yt-dlp is completely compatible with youtube-dl. Should I just fix the typo or is that too much info in this location?

Reply:
fixing the typos is ok

Reply:
Alright, done!